### ISSUE TITLE
Google Analytics for Packages not working

### ISSUE THREAD

**Issue by enkelmedia** [umbraco/OurUmbraco#475]
When editing a Package we have the option to add a "Google Analytics-code", this used to work but some update might have broken this.

When loading a Package Page with a GA-code set I get this error in the console:

Uncaught ReferenceError: ga is not defined

<img width="420" alt="Skärmavbild 2019-09-27 kl  08 03 20" src="https://user-images.githubusercontent.com/1782524/65745842-580b6400-e0fd-11e9-9204-8e4dfd64f8b9.png">

I'm guessing that maybe the ga-script was removed from the header in favor of GTM, the code in the view that triggers the tracking looks like this:

```
<script type="text/javascript">
      ga('create', 'UA-xxxxxxxx-x', 'auto', { 'name': 'PackageMaker' });
      ga('PackageMaker.send', 'pageview');
</script>
```

And it's not working since ga is not defined when the code is executed, probably that GTM is used to load Google Analytics async. If I trigger the code from the console after the page has been loaded it works just fine which indicates that ga is loaded later than the trigger code.

There's probably a number of ways to solve this but a simple solution would be to trigger the script with a timeout 

```
<script type="text/javascript">
    setTimeout(function(){ 
         ga('create', 'UA-xxxxxxxx-x', 'auto', { 'name': 'PackageMaker' })
         ga('PackageMaker.send', 'pageview');
      }, 1000);
</script>
```

Would you like a pull request with this solution or would you prefer some other solution? Could also be a timeout that triggers it self until ga is loaded to be more safe over the 1000 ms that is used right now. 

This is where the code lives: https://github.com/umbraco/OurUmbraco/blob/695e7d56231abab4cbb3caba7a7e666df0f8e445/OurUmbraco.Site/Views/Partials/Projects/ProjectDetails.cshtml#L475

**Comment by nul800sebastiaan** [comment]
Aha.. yeah I didn't think about this one when we update the GA scripts to use TagManager. We'll have to look at this!

**Comment by enkelmedia** [comment]
I guess that the triggers thats implemented as "on-click" on a hrefs tags would still work as they are fired when ga is loaded =D

Let me know if I can help out =D

**Comment by nul800sebastiaan** [comment]
Thanks @enkelmedia - we've fixed it in PR #483 - I'll roll this out later today.
